pkg/extensions/sync/on_demand.go

```diff
@@ -156,7 +156,7 @@ func syncOneImage(imageChannel chan error, cfg Config, storeController storage.S
 
 				// is notary signature
 				if isArtifact {
-					err = syncNotarySignature(httpClient, storeController, *regURL, repo, tag, log)
+					err = syncNotarySignature(httpClient, storeController, *regURL, repo, repo, tag, log)
```

Comment:
Here localRepo and remoteRepo is same or we want same repo name in destination that's why using repo name for both (localRepo and remoteRepo) ?

Reply:
I think he implemented this feature only for periodically sync and not for on demand, yet.